languages/translations/en.json

```diff
@@ -41,7 +41,8 @@
                 "ox": "Keep left",
                 "xox": "Keep in the middle",
                 "oxo": "Keep left or right"
-            }
+            },
+            "ceremonialNamesToAvoid": ["freeway", "expressway", "highway", "fwy", "hwy", "expy"]
```

Comment:
It’s unlikely that Transifex would be able to handle an array like this well. Even if it does present the array to translators, they’ll have to provide exactly six words to avoid, no more, no less.

We could put this array in the [overrides](https://github.com/Project-OSRM/osrm-text-instructions/blob/c2c8b2e90964d4508d5e6bbcc60f0c7cab5401b2/languages/overrides/de.js) file, but that would shut Transifex entirely out of the process. Or we could join the list together into a single, comma-delimited string, similar to the approach we took in mapbox/mapbox-navigation-ios#254.